samples/java_springboot/45.state-management/src/main/java/com/microsoft/bot/sample/statemanagement/Application.java

```diff
@@ -3,9 +3,10 @@
 
 package com.microsoft.bot.sample.statemanagement;
 
-import com.microsoft.bot.builder.Bot;
-import com.microsoft.bot.builder.ConversationState;
-import com.microsoft.bot.builder.UserState;
+import com.microsoft.bot.azure.CosmosDbPartitionedStorage;
+import com.microsoft.bot.azure.CosmosDbPartitionedStorageOptions;
+import com.microsoft.bot.azure.blobs.BlobsStorage;
+import com.microsoft.bot.builder.*;
```

Comment:
We need to replace this import with individuals imports, right?

Reply:
Thanks for that